Dockerfile

```diff
@@ -57,6 +54,7 @@ RUN apt-get update && \
         mysql-client \
         poppler-utils \
         sudo \
+        vim \
```

Comment:
> Not possible because the images are mounted read-only by default.

IIRC I've been modifying stuff like this: just "login" to a container as root, remount in "rw" mode, and install what is wanted.